src/framework/preferences/css/stylus/utils/Helpers.styl

```diff
@@ -21,3 +21,23 @@ linear-gradient() {
         s('linear-gradient(%s)', content);
     }
 }
+
+
+// Mixin for common styles of the "thumb" (slider control)
+// used by both the jQuery and native HTML sliders
+slider-thumb-common() {
+    height: 1.5em;
+    width: 1.5em;
+    background-color: #fff;
+    border: 1px solid #b3b3b3;
+    border-radius: 2em;
+    box-shadow: 4px 2px 3px rgba(0,0,0,0.3);
```

Comment:
@jhung along the lines of adaptive components, I wonder as well about whether or not the `box-shadow` shouldn't be removed entirely from the thumb in the various UIO contrast themes. Does it potentially interfere the accessibility of the contrast theme?

It's quite visible in some contrast themes and nearly invisible in others - I see looking at `Themes.styl` that we remove `box-shadow` for some of the other components. Two screenshots for comparison (the black-on-white and yellow-on-black themes). 

![image](https://cloud.githubusercontent.com/assets/5279051/20015301/57dd7e76-a291-11e6-9e27-87065b60b458.png)

![image](https://cloud.githubusercontent.com/assets/5279051/20015314/65167e8a-a291-11e6-87d5-7bcb679e47cf.png)